repository/repository.go

```diff
@@ -25,6 +25,15 @@ var repoToOrg = map[string]string{
 	"k3s":  "k3s-io",
 }
 
+// stripBackportTag returns a string with a prefix backport tag removed
+func stripBackportTag(s string) string {
+	if strings.Contains(s, "elease") {
+		s = strings.Split(s, "]")[1]
+	}
+	s = strings.Trim(s, " ")
+	return s
```

Comment:
Looks like we're missing the Title call to make sure the first character is capitalized.

Reply:
strings.Title capitalizes every word, not just the first character. I added a capitalization helper that can be used inside templates in #153

Reply:
I hadn't noticed that behavior before. Cool. Thanks!\